### ISSUE TITLE
Import config into web installer

### ISSUE THREAD

**Issue by dadofsambonzuki** [LightningPiggy/lightning-piggy#18]
It would be great to be able to import config (wifi credentials, LNBits server, etc.) into the web installer so that we can upgrade the firmware without having to re-enter the details.

**Comment by ThomasFarstrike** [comment]
True! Although it would also be solved by https://github.com/LightningPiggy/lightning-piggy/issues/21

Perhaps it can even be incorporated into that one...

**Comment by ThomasFarstrike** [comment]
As #21 is done, the configuration is preserved between web installer runs, and it can also be easily modified using the Access Point configuration mode, so this can be closed!
